CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added side panel for lineage graph
 ### Changed
 - Tweaked color scheme and icons
+- Added a new service responsible for constructing a graph 
```

Comment:
You added a note to already released version. Move it to `Unreleased`. Note that master has a new commit, so rebase before starting.